### PR TITLE
chore: change trivial logs from INFO to DEBUG

### PR DESCRIPTION
**Pull request by ccx1024cc**
### DESCRIPTION
Working with multiple layers, snapshotter produces trivial logs.

### REFACT
This patch changes two logs from INFO to DEBUG.

### CODE REVIEW

**Comment by yqleng1987** [comment]
@ccx1024cc , a new test job has been submitted. Please wait in patience.

**Review by jiangliu — APPROVED** [review]

**Comment by yqleng1987** [comment]
@ccx1024cc , The CI test is completed, please check result:
<table>
    <tr>
        <th colspan="2">Test Case</th><th>Test Result</th>
    </tr>
    <tr>
        <td colspan="2">merge-target-branch</td><td><a href=https://tone.openanolis.cn/ws/nrh4nnio/test_result/6793>:white_check_mark:SUCCESS</a></td>
    </tr>
    <tr>
        <td colspan="2">build-docker-image</td><td><a href=https://tone.openanolis.cn/ws/nrh4nnio/test_result/6793>:white_check_mark:SUCCESS</a></td>
    </tr>
    <tr>
        <td colspan="2">compile-nydus</td><td><a href=https://tone.openanolis.cn/ws/nrh4nnio/test_result/6793>:white_check_mark:SUCCESS</a></td>
    </tr>
    <tr>
        <td colspan="2">compile-ctr-remote</td><td><a href=https://tone.openanolis.cn/ws/nrh4nnio/test_result/6793>:white_check_mark:SUCCESS</a></td>
    </tr>
    <tr>
        <td colspan="2">compile-nydus-snapshotter</td><td><a href=https://tone.openanolis.cn/ws/nrh4nnio/test_result/6793>:white_check_mark:SUCCESS</a></td>
    </tr>
    <tr>
        <td colspan="2">start-nydus-snapshotter-config-containerd</td><td><a href=https://tone.openanolis.cn/ws/nrh4nnio/test_result/6793>:white_check_mark:SUCCESS</a></td>
    </tr>
    <tr>
        <td colspan="2">run-container-with-nydus-image</td><td><a href=https://tone.openanolis.cn/ws/nrh4nnio/test_result/6793>:white_check_mark:SUCCESS</a></td>
    </tr>
</table>

Congratulations, your test job passed!

File: rafs/src/metadata/md_v5.rs [hunk 1]
```diff
@@ -25,7 +25,7 @@ impl RafsSuper {
         self.meta.chunk_size = sb.block_size();
         self.meta.flags = RafsSuperFlags::from_bits(sb.flags())
             .ok_or_else(|| einval!(format!("invalid super flags {:x}", sb.flags())))?;
-        info!("rafs superblock features: {}", self.meta.flags);
+        debug!("rafs superblock features: {}", self.meta.flags);
```
Comment (changweige) [inline rafs/src/metadata/md_v5.rs:28]: Um... I personally think it is helpful to investigate problems with these feature flags

Comment (ccx1024cc) [reply rafs/src/metadata/md_v5.rs:28]: > Um... I personally think it is helpful to investigate problems with these feature flags

U r right. Just Remove unpacker's log

**Comment by yqleng1987** [comment]
@ccx1024cc , your pull request has been updated, but no change is detected in the code.

**Comment by yqleng1987** [comment]
@ccx1024cc , your pull request has been updated. A new test job will be submitted. Please wait in patience.

**Comment by yqleng1987** [comment]
@ccx1024cc , your test job has passed, and no need to test again.

**Pull request merged**